### PR TITLE
Logira draft

### PR DESCRIPTION
**Pull request by LoganGirard**
Tore this out into it's own module, basically re-using most of the code, just with different selectors. There are new params that we need to support (ban counts) I'm assuming the discord bot handles this by default. Let's push. 

Also added prettier module for formatting, can revert if we don't like it.

### CODE REVIEW

File: drafting-utils.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+const puppeteer = require("puppeteer");
+
+// Delay Function
+const delay = (ms) => new Promise((res) => setTimeout(res, ms));
+
+configs = {
+  BLUE_5_RED_5: "https://draftlol.dawe.gg?opts=InAiOjQ1LCJiIjo0NQ",
```
Comment (teajason) [inline drafting-utils.js:7]: A quick comment wouldn't hurt that explains that the numbers here tell you how many bans that team gets.

File: index.js [hunk 2]
```diff
@@ -1,366 +1,413 @@
...
-			
-			let blue = await page.$('#blue_team_name')
-			let red = await page.$('#red_team_name')
-			let match = await page.$('#match_name')
+  // Command handler
+  switch (command) {
+    case "autoassign":
+      message.reply("Autoassign method [not implemented yet]");
+      break;
+    case "fa":
```
Comment (teajason) [inline index.js:43]: Out of curiosity... What does case "FA" mean? Or FF for that matter?

Comment (justin8502) [reply index.js:43]: "fa" is soon to be deprecated - it adds the "Free Agent" role to an individual on discord. "ff" is the equivalent of a "!help" command, changed to ff due to other bots in the server already responding to "!help".

Comment (teajason) [reply index.js:43]: ohh

**Pull request merged**